shared/login/relogin/container.tsx

```diff
@@ -115,7 +118,8 @@ export default Container.connect(
     onLogin: dispatchProps.onLogin,
     onSignup: dispatchProps.onSignup,
     onSomeoneElse: dispatchProps.onSomeoneElse,
-    selectedUser: stateProps.selectedUser,
+    resetBannerUser: ownProps.resetSuccess ? stateProps._autoresetUser : null,
+    selectedUser: ownProps.resetSuccess ? stateProps._autoresetUser : stateProps.selectedUser,
```

Comment:
This shows up empty in the dropdown
<img width="965" alt="image" src="https://user-images.githubusercontent.com/11968340/66322045-85bb8e80-e8ef-11e9-8787-2ee4dca01045.png">

Maybe it needs to be added to `_users`

Reply:
I added it to users, and then it failed when you clicked it because it needs a provision action and not a login action. I removed it but forgot to remove it here.

Reply:
We could instead take you to the regular login screen.